### PR TITLE
Implement `OperationValue` variants for all relation types

### PR DESCRIPTION
**Pull request by sandreae**
- Implement `OperationValue` variants for all `Relation` types
- Remove now unused `OperationValueRelation*` type

## 📋 Checklist

- [x] Add tests that cover your changes
- [x] Add this PR to the _Unreleased_ section in `CHANGELOG.md`
- [x] Link this PR to any issues it closes
- [x] New files contain a SPDX license header
- [x] Check if descriptions and terminology match `handbook` content (and visa-versa)

### CODE REVIEW

**Review by adzialocha — APPROVED** [review]
Super! I really like it! Looks clean :+1:

**Comment by sandreae** [comment]
Yeh, I was mainly removing code, which felt good.

**Comment by sandreae** [comment]
> Is there a handbook PR for this which covers this change?

Aha, not yet.

File: p2panda-js/src/wasm.test.ts [hunk 1]
```diff
@@ -99,6 +99,22 @@ describe('WebAssembly interface', () => {
         'Invalid integer value',
       );
 
+      expect(() => fields.add('contact', 'relation', [TEST_HASH])).toThrow(
+        'Invalid object',
```
Comment (cafca) [inline p2panda-js/src/wasm.test.ts:103]: I think this error message could be improved

**Comment by sandreae** [comment]
Related PR for the handbook: https://github.com/p2panda/handbook/pull/158

**Comment by sandreae** [comment]
haaaa? Now test cov decreases :sob:

File: p2panda-js/src/wasm.test.ts (hunk 1 above)
Comment (cafca) [reply p2panda-js/src/wasm.test.ts:103]: https://github.com/p2panda/p2panda/issues/262

**Review by cafca — APPROVED** [review]

**Comment by cafca** [comment]
> haaaa? Now test cov decreases 😭

It's because this PR removes lines that were previously tested so the relative coverage decreases.

File: p2panda-js/src/wasm.test.ts (hunk 1 above)
Comment (sandreae) [reply p2panda-js/src/wasm.test.ts:103]: Yeh, very true, you wanna do that as part of this PR or a separate one?

Comment (cafca) [reply p2panda-js/src/wasm.test.ts:103]: No separately!

**Comment by sandreae** [comment]
But it went up before???

**Comment by sandreae** [comment]
codecov is so fickle ;-p

File: p2panda-js/src/wasm.test.ts (hunk 1 above)
Comment (sandreae) [reply p2panda-js/src/wasm.test.ts:103]: I see in the issue I think you wanna do a wider revamp of the error messages :+1:

**Pull request merged**